scripts/maintenance/migrate_project/Makefile

```diff
@@ -1,23 +1,36 @@
-.DEFAULT_GOAL := build
+.DEFAULT_GOAL := help
 
 SHELL := /bin/bash
 
-CFG_NAME	:= cfg.ignore.json
+CFG_NAME	:= cfg.json
 REPO_DIR	:= $(CURDIR)/../../..
 IMAGE_NAME	:= migrate-utils
 TAG			:= latest
 
-build:
+.PHONY: help
+
+help: ## help on rule's targets
+ifeq ($(IS_WIN),)
+	@awk --posix 'BEGIN {FS = ":.*?## "} /^[[:alpha:][:space:]_-]+:.*?## / {printf "\033[36m%-20s\033[0m %s\n", $$1, $$2}' $(MAKEFILE_LIST)
+else
+	@awk --posix 'BEGIN {FS = ":.*?## "} /^[[:alpha:][:space:]_-]+:.*?## / {printf "%-20s %s\n", $$1, $$2}' $(MAKEFILE_LIST)
+endif
+
+.PHONY: build
+build: ## Builds docker container for the migration. Run this first.
 	docker build -t ${IMAGE_NAME} --file $(PWD)/Dockerfile $(REPO_DIR)
```

Comment:
good practice: please use verbose options in any written script.

Reply:
Sorry I do not understand, this is currently not actionable for me:

- A google search for `verbose docker build` gives no reasonable results: https://www.google.com/search?client=firefox-b-d&q=verbose+docker+builld

What should be verbose here, could you point me to an example for how you need this makefile to look?